terraform/staging/main.tf

```diff
@@ -70,9 +46,12 @@ module "application" {
   database_port                        = local.database_port
   db_security_group_id                 = module.database.rds_security_group_id
   ecr_repository_url                   = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/core"
-  ecs_task_cpu                         = 512
-  ecs_task_desired_count               = 2
-  ecs_task_memory                      = 1024
+  ecs_app_task_cpu                     = 512
+  ecs_app_task_desired_count           = 2
+  ecs_app_task_memory                  = 1024
+  ecs_sidekiq_task_cpu                 = 1024
+  ecs_sidekiq_task_desired_count       = 1

```

Comment:
I think we want a single ECS task for sidekiq rather than multiple (otherwise each task will try and run the same jobs at the same time). If for some reason the task starts to hit some cpu / memory threshold, I think we'll want to scale vertically rather than horizontally (which will be different scaling behaviour to the application task)

Reply:
I'm not sure there's a ticket anywhere that covers auto-scaling setup. We could update [CLDC-2617](https://dluhcdigital.atlassian.net/browse/CLDC-2617) for this or make one?